calibrate.py

```diff
@@ -340,6 +379,12 @@ def capture_images(self):
                     frame_list.append(small_frame)
 
                     if captured_left and captured_right:
+                        print("Images captured-->")
```

Comment:
These could be on the same line: `print("Images captured-->"+str(self.images_captured))`

Reply:
even `print(f"Images captured --> {self.images_captured}")`

Reply:
got it